assets/queries/cloudFormation/ecs_cluster_at_rest_encryption/query.rego

```diff
@@ -0,0 +1,21 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].Resources
+  elem := resource[key]
+  elem.Type == "AWS::ECS::Service"
+  clustername := elem.Properties.Cluster
+  taskdefinitionkey := elem.Properties.TaskDefinition
+  taskDefinition := resource[taskdefinitionkey]
+  
+  count(taskDefinition.Properties.ContainerDefinitions)>0
+  taskDefinition.Properties.Volumes[j].EFSVolumeConfiguration.TransitEncryption == "DISABLED"
```

Comment:
Since we want to ensure encryption, shouldn't we also check if the taskdefinition actually exists? If it doesn't, the query won't capture it and it will unduly pass

Reply:
Resolved